packages/suite/src/components/suite/labeling/MetadataLabeling/definitions.ts

```diff
@@ -10,13 +10,15 @@ export interface DropdownMenuItem {
 
 export interface Props {
     defaultVisibleValue?: ReactNode;
+    defaultEditableValue?: string;
     payload: MetadataAddPayload;
     dropdownOptions?: DropdownMenuItem[];
     isDisabled?: boolean;
     // override default onSubmit logic
     onSubmit?: (value: string | undefined) => void;
     // override default behavior of metadata labeling element visible only on hover
     visible?: boolean;
+    placeholder?: string;
```

Comment:
As I went through multiple designs, I introduced a placeholder for the auto-scaling input as some point. After few iteration with Jakub it was decided that we wont use it now but we probably use it when we do redesign of the whole labeling. So I kept it here as it is kinda standard prop for any input and because `AutoScalingInput` component supports that.